GPU/TPCFastTransformation/TPCFastTransform.h

```diff
@@ -78,8 +101,17 @@ class TPCFastTransform : public FlatObject
   /// Assignment operator: disabled to avoid ambiguity. Use cloneFromObject() instead
   TPCFastTransform& operator=(const TPCFastTransform&) CON_DELETE;
 
-  /// Destructor
+/// Destructor
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE)
+  ~TPCFastTransform()
+  {
+    if (mCorrectionSlow) {
+      delete mCorrectionSlow;
+    }
```

Comment:
delete checks for a nullptr and does nothing on it, you don't need to check yourself

Reply:
Thanks, I did not know that.